source/includes/usage-examples/retrieve-code-examples.kt

```diff
@@ -0,0 +1,57 @@
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.kotlin.client.*
```

Comment:
Normally, I wouldn't recommend `*` imports but given the example is pretty open, I think its ok for brevity.